defaults/main.yml

```diff
@@ -17,9 +17,7 @@ admin_sudoers: True
 #moreusers:
 #  - {name: someotheruser1, state: 'present', uid: 6001, shell: "{{adminshell}}" }
 
-admin_list_of_user_lists:
-  - "{{ adminusers | default([]) }}"
-  - "{{ moreusers | default([]) }}"
```

Comment:
This is a list of list. -> list.

Reply:
However, the variable name still says it's a list of lists

Reply:
That would be a breaking change. I can't control if users are using the variable `admin_list_of_user_lists`